app/views/components/gds/saveButtons.scala.html

```diff
@@ -0,0 +1,25 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@this(
+        saveAndContinue: saveAndContinue
+)
+
+@(saveAndContinueKey: String = "site.save_and_continue", saveAsDraftKey: String = "site.save_and_come_back_later")(implicit messages: Messages)
+
+@saveAndContinue(saveAndContinueKey)
+<br>
+@saveAsDraft(saveAsDraftKey)
```

Comment:
Is this working as expected? `saveAsDraft` is not injected.

Reply:
It is.  I think it must be because saveAsDraft does not contain any injected GDS components.